### PR TITLE
[Keymap] Reviung41_Ciutadellla

### PR DESCRIPTION
**Pull request by ciutadellla**
Add Personal Keymap for Reviung41 Keyboard

### CODE REVIEW

**Comment by ciutadellla** [comment]
@skullydazed @drashna Hi, This is my first Keymap. Can you please help me with the approval? Amazing project 💯

File: keyboards/reviung41/keymaps/ciutadellla/config.h [hunk 1]
```diff
@@ -0,0 +1,14 @@
+
```
Comment (drashna) [inline keyboards/reviung41/keymaps/ciutadellla/config.h:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/reviung41/keymaps/ciutadellla/config.h (hunk 1 above)
Comment (ciutadellla) [reply keyboards/reviung41/keymaps/ciutadellla/config.h:16]: @drashna Done ;)

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**